schema.sql

```diff
@@ -32,3 +32,23 @@ DROP species;
 ALTER TABLE animals ADD COLUMN species_id INT REFERENCES species(id);
 
 ALTER TABLE animals ADD COLUMN owners_id INT REFERENCES owners(id);
+
+----------------------Day-4-Project--------------------------------
+
+CREATE TABLE vets (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name VARCHAR(100),
+    age INT CHECK (age > 0 AND age < 130),
+    date_of_graduation DATE,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE specializations (
+species_id INT REFERENCES species(id),
+vets_id INT REFERENCES vets(id)
+);
+
+CREATE TABLE visits (
+animals_id INT REFERENCES animals(id),
+vets_id INT REFERENCES vets(id)
+);
```

Comment:
- [ ] in the visits table there should be a column with date included as well

Reply:
@mikethreels Thank you for your review, the code has been corrected accordingly.